By default, ~/.antiword is used to store configuration files. These changes
allow to use a custom directory so we can use the EasyBuild install directory
for it
Author: Steven Vandenbrande
--- antiword.h.orig	2025-09-19 13:06:02.013082000 +0200
+++ antiword.h	2025-09-19 13:06:36.158834000 +0200
@@ -225,7 +225,9 @@
 #define FONTNAMES_FILE		"fontnames"
 #else	/* All others */
 #define GLOBAL_ANTIWORD_DIR	"/usr/share/antiword"
+#ifndef ANTIWORD_DIR
 #define ANTIWORD_DIR		".antiword"
+#endif
 #define FONTNAMES_FILE		"fontnames"
 #endif /* __dos */
 /* The names of grouped mapping files */
--- options.c.orig	2025-09-19 13:45:11.541933000 +0200
+++ options.c	2025-09-19 13:45:44.335772000 +0200
@@ -224,8 +224,8 @@
 	    sizeof(ANTIWORD_DIR) -
 	    2 * sizeof(FILE_SEPARATOR)) {
 		sprintf(szMappingFile,
-			"%s" FILE_SEPARATOR ANTIWORD_DIR FILE_SEPARATOR "%s%s",
-			szHome, szLeafname, szSuffix);
+			FILE_SEPARATOR ANTIWORD_DIR FILE_SEPARATOR "%s%s",
+			szLeafname, szSuffix);
 		DBG_MSG(szMappingFile);
 		pFile = fopen(szMappingFile, "r");
 		if (pFile != NULL) {
